GA4: Page call support (#562)
rudderlabs/rudderstack-docs: destinations/analytics/google-analytics-4/google-analytics-4-cloud-mode.mdx

});
```

## Page

The [`page`](https://www.rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-spec/page/) call lets you record your website's page views with any additional relevant information about the viewed page. 

> RudderStack maps the `page` call to a `page_view` event (by default) and passes it to Google Analytics 4 as a custom event. As mentioned above, the cloud mode supports both the `gtag` and `firebase` ways for tagging in websites. 

Reviewer: ```suggestion
RudderStack maps the `page` call to a `page_view` event by default, and passes it to Google Analytics 4 as a custom event. 

As mentioned in the [`track`](#track) section above, the RudderStack cloud mode supports both the `gtag` and `firebase` methods for tagging in websites. 
```

Google Analytics 4 has some reserved event, parameter, and user property names that cannot be used. Refer to the [Measurement Protocol (Google Analytics 4)](https://developers.google.com/analytics/devguides/collection/protocol/ga4/reference?client_type=gtag#reserved_names) guide for a complete list of reserved names.

### How do I obtain the `app_instance_id`?

> The `app_instance_id` can be retrieved through the Firebase SDK depending on the platform on which the SDK is installed: 

Reviewer: ```suggestion
You can retrieve the `app_instance_id` through the Firebase SDK depending on the platform where the SDK is installed: 
```

Google Analytics 4 has some reserved event, parameter, and user property names that cannot be used. Refer to the [Measurement Protocol (Google Analytics 4)](https://developers.google.com/analytics/devguides/collection/protocol/ga4/reference?client_type=gtag#reserved_names) guide for a complete list of reserved names.

### How do I obtain the `app_instance_id`?

The `app_instance_id` can be retrieved through the Firebase SDK depending on the platform on which the SDK is installed: 

- Android: `getAppInstanceId()`
- Kotlin: `getAppInstanceId()`
- Swift: `appInstanceID()`
- Objective-C: `appInstanceID`
- C++: `GetAnalyticsInstanceId()`

> - Unity: `GetAnalyticsInstanceIdAsync()`

Reviewer 2: Each of those functions() has a hyperlink stating how to carry out those operations. Should we mention these or just provide ref to https://developers.google.com/analytics/devguides/collection/protocol/ga4/reference?client_type=firebase#payload_post_body?

Author: makes sense, Akash. I have directly added the links to each platform and removed GA4 doc reference.
